### ISSUE TITLE
Make CloudConnection instances EventEmitter instances

### ISSUE THREAD

**Issue by hello-smile6** [webdev03/meowclient#18]
It would be helpful to be able to react to cloud variable changes.

**Comment by webdev03** [comment]
React to cloud variable changes? I was thinking of adding that. But check this out:
```js
CloudConnection.connection.on("message", (e) => {
      if (!e) return;
      for (const message of e.toString().split("\n")) {
        const obj = JSON.parse(message || '{"method": "err"}');
        if (obj.method == "set") {
          // obj.name is the name, obj.value is value
        }
      }
    });
```

**Comment by hello-smile6** [comment]
> React to cloud variable changes? I was thinking of adding that. But check this out:
> 
> ```js
> CloudConnection.connection.on("message", (e) => {
>       if (!e) return;
>       for (const message of e.toString().split("\n")) {
>         const obj = JSON.parse(message || '{"method": "err"}');
>         if (obj.method == "set") {
>           // obj.name is the name, obj.value is value
>         }
>       }
>     });
> ```
> 
> I'll add a better way to do that (callback hook probably) soon

Being able to use addEventListener would be easier.

**Comment by hello-smile6** [comment]
https://stackoverflow.com/a/39145058
Working on implementation.

**Comment by hello-smile6** [comment]
Closed by #19
